apis/fluentd/v1alpha1/plugins/input/monitor_agent.go

```diff
@@ -0,0 +1,17 @@
+package input
+
+// The in_monitor_agent Input plugin exports Fluentd's internal metrics via REST API.
+type MonitorAgent struct {
+	// The port to listen to.
+	Port *int64 `json:"port,omitempty"`
+	// The bind address to listen to.
+	Bind *string `json:"bind,omitempty"`
+	// If you set this parameter, this plugin emits metrics as records.
+	Tag *string `json:"tag,omitempty"`
+	// The interval time between event emits. This will be used when "tag" is configured.
+	EmitInterval *int64 `json:"emit_interval,omitempty"`
```

Comment:
All the field of CRDs in fluent operator use  camelCase, should be `emitInterval` instead of `emit_interval`, while the config files items should be consistent with fluentbit or fluentd conversion which is `emit_interval`.
https://github.com/fluent/fluent-operator/blob/master/apis/fluentd/v1alpha1/plugins/output/opensearch.go#L20 opensearch output plugin is an example

Reply:
Fixed in https://github.com/fluent/fluent-operator/pull/967/commits/a6fcd16d51a87ca170636d8cd0c11aa4b895417f